Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "ha_api"
 description = "This is a library for connecting to the Home Assistent API"
-version = "0.1.0"
+version = "1.0.0-kylep-add-as-mobile-app-SNAPSHOT"
```

Comment:
The problem with this is that it "Pollutes the versions available on crates.io. I have no problem putting beta version on there but it means that updates here will need to be approved before updates to halcyon can be reviewed.

Reply:
Also, this particular scheme is not supported by cargo [link to docs](https://doc.rust-lang.org/cargo/reference/resolver.html#version-metadata)